src/airport.js

```diff
@@ -1 +1,79 @@
+class Airport {
+    #capacity
```

Comment:
Used private fields to encapsulate data and prevent direct access from outside the class.